generators/client/templates/src/test/javascript/spec/app/account/reset/finish/_reset.finish.controller.spec.js

```diff
@@ -54,7 +54,7 @@ describe('Controller Tests', function() {
 
             // then
             expect(MockTimeout).toHaveBeenCalledWith(jasmine.any(Function));
-            expect(MockAngular.element).toHaveBeenCalledWith('[ng-model="vm.resetAccount.password"]');
```

Comment:
After I updated  `_reset.finish.controller.js` and `_reset.request.controller.js` to use ID instead of attribute, it caused the test to failed with the following errors : 

 
reset.finish.controller.spec.js :
`Expected spy MockAngular.element to have been called with [ '[ng-model="vm.resetAccount.password"]' ] but actual calls were [ '#password' ].`
 
reset.request.controller.spec.js :
`Expected spy MockAngular.element to have been called with [ '[ng-model="resetAccount.email"]' ] but actual calls were [ '#email' ]. `


I tried to use following instead of `[ '[ng-model="vm.resetAccount.password"]' ]` and `[ ng-model="resetAccount.email"]' ]`   but they didn't work : 

```
$('#password')
$('#password').val()
```